### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by htcml** [JKCooper2/gym-bandits#4]
I followed the instruction to install the package and run the sample code and got the following error. I use Python 3.7.3.

git clone git@github.com:JKCooper2/gym-bandits.git
cd gym-bandits
pip install .

import gym_bandits
env = gym.make("BanditTenArmedGaussian-v0") # Replace with relevant env

Error:
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-3-2f7d68eb2ef5> in <module>
----> 1 import gym_bandits
      2 env = gym.make("BanditTenArmedGaussian-v0") # Replace with relevant env

~/anaconda3/lib/python3.7/site-packages/gym_bandits/__init__.py in <module>
     24         entry_point='gym_bandits:{}'.format(environment[0]),
     25         timestep_limit=1,
---> 26         nondeterministic=True,
     27     )

~/anaconda3/lib/python3.7/site-packages/gym/envs/registration.py in register(id, **kwargs)
    151 
    152 def register(id, **kwargs):
--> 153     return registry.register(id, **kwargs)
    154 
    155 def make(id, **kwargs):

~/anaconda3/lib/python3.7/site-packages/gym/envs/registration.py in register(self, id, **kwargs)
    145         if id in self.env_specs:
    146             raise error.Error('Cannot re-register id: {}'.format(id))
--> 147         self.env_specs[id] = EnvSpec(id, **kwargs)
    148 
    149 # Have a global registry

TypeError: __init__() got an unexpected keyword argument 'timestep_limit'

**Comment by spencerkraisler** [comment]
I got this error too. I went into __init__.py and deleted line 25 that said "timestep_limit = 1". This got rid of the error for me. Remember to go into the repo and re-install it in order for it to update your changes. 

A warning pops up after as well, but it all still worked for me. To rid of the error, I went into bandit.py and changed the method names "_step" and "_reset" to "step" and "reset" respectively. It seems those were deprecated.
